src/CommandingOfficers/Bear_Bull.java

```diff
@@ -164,11 +149,19 @@ protected void perform(MapMaster gameMap)
     @Override
     public GameEventQueue getEvents(MapMaster gameMap)
     {
+      Collection<Unit> victims = findVictims(gameMap);
+
       // Damage is dealt after swapping D2Ds so it's actually useful to Bear
       GameEventQueue powerEvents = new GameEventQueue();
-      powerEvents.add( (map) -> COcast.liquidateMassDamage = true ); // Collect money instead of ability energy
-      powerEvents.add(new MassDamageEvent(findVictims(gameMap), DOWNUPTURN_LIQUIDATION, false));
-      powerEvents.add( (map) -> COcast.liquidateMassDamage = false );
+      powerEvents.add(new MassDamageEvent(COcast, victims, DOWNUPTURN_LIQUIDATION, false));
+
+      int valueDrained = 0;
+      for( Unit victim : victims )
+      {
+        valueDrained += (Math.min(DOWNUPTURN_LIQUIDATION, victim.getHP()) * victim.model.getCost()) / victim.model.maxHP;
```

Comment:
```suggestion
        valueDrained += (Math.min(DOWNUPTURN_LIQUIDATION, victim.getHP()-1) * victim.model.getCost()) / victim.model.maxHP;
```

Reply:
I thought about that, but you can actually drain a unit to 0.1 HP, and that's pretty niche anyway.
May as well give you the small benefit.

Reply:
Really? Map damage is normally in whole increments. I guess why not though if the power actually drains to 0.1.

Reply:
Map damage goes down to 0.1 HP on cart and I believe I've mirrored that here.